src/assets/fabricator/styles/components/layout.css

```diff
@@ -0,0 +1,67 @@
+.f-Layout {
+  height: 100%;
+}
+
+.f-Layout.f-is-active {
```

Comment:
Not sure the `f-` namespace is needed on the states, since they shouldn't have any independent styles. The requirement of them only being used in conjunction with other classes prevents collision, I think.